src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -994,6 +994,17 @@ def load_pem_public_key(self, data):
             else:
                 self._handle_key_loading_error()
 
+    def load_pem_parameters(self, data):
```

Comment:
This (and load_der) both need to be added to their respective interfaces and the interface docs.